### PR TITLE
Added matched_queries to document

### PR DESCRIPTION
**Pull request by alxpy**

### CODE REVIEW

**Comment by popravich** [comment]
any…
* [ ] …public interface for this private option?
* [ ] …tests?

File: elasticmagic/document.py [hunk 1]
```diff
@@ -141,6 +141,7 @@ def __init__(self, _hit=None, _result=None, **kwargs):
         self._index = self._type = self._id = self._score = None
         self._hit_fields = None
         self._highlight = None
+        self.matched_queries = None
```
Comment (anti-social) [inline elasticmagic/document.py]: Think field should be private and there should be method like `get_matched_queries` or `get_matched_query_names`.

**Comment by alxpy** [comment]
@anti-social, fixed.

**Comment by anti-social** [comment]
Thanks.

**Pull request merged**